tests/rest/client/test_rooms.py

```diff
@@ -832,6 +832,24 @@ async def user_may_join_room(
 
         self.assertEqual(join_mock.call_count, 0)
 
+        async def user_may_join_room_tuple(
+            mxid: str,
+            room_id: str,
+            is_invite: bool,
+        ) -> Tuple[Codes, dict]:
+            return (Codes.CONSENT_NOT_GIVEN, {})
+
+        join_mock.side_effect = user_may_join_room_tuple
+
+        channel = self.make_request(
+            "POST",
+            "/createRoom",
+            {},
+        )
+        self.assertEqual(channel.code, 200, channel.json_body)
```

Comment:
Let's also add some checks to make sure we get the correct code and fields here.

Reply:
Well, I've added such checks in other places, but in this test/at this line, we're asserting that the spam-checker is bypassed when creating a room, so I'm not sure what you want me to look for.

Reply:
Bleh, you're right, sorry, I misread.